tap_nice_incontact/schemas/qm_workflows.json

```diff
@@ -0,0 +1,133 @@
+{
+  "type": ["null", "object"],
+  "additionalProperties": false,
+  "properties": {
+    "tenantName": {
+      "type": ["null", "string"]
+    },
+    "agentId": {
+      "type": ["null", "string"]
+    },
+    "type": {
+      "type": ["null", "string"]
+    },
+    "evaluationFlowTrigger": {
+      "type": ["null", "string"]
+    },
+    "workflowStatus": {
+      "type": ["null", "string"]
+    },
+    "autoAcknowledged": {
+      "type": ["null", "string"]
+    },
+    "workflowState": {
+      "type": ["null", "string"]
+    },
+    "workflowId": {
+      "type": ["null", "string"]
+    },
```

Comment:
all of these properties have null in them. are we sure that they can all be null? might break if all of them are null or have update the wrong piece of data since the composite key is workflowId, lastUpdated, and submissionDate. I would imagine that workflowID is the unique identifier.

Reply:
Also updated it to use `startDate` instead of `submissionDate`